int16 numObstacles
int16 pathSize
geometry_msgs/Vector3[] obstacles
geometry_msgs/Vector3[] path